src/pybind/mgr/dashboard/frontend/src/app/ceph/block/mirroring/image-list/image-list.component.html

```diff
@@ -1,5 +1,7 @@
-<tabset>
-  <tab heading="Issues"
+<tabset stateId="block-mirroring-images">
+  <tab cdTabStateful
+       stateId="issues"
+       heading="Issues"
```

Comment:
Can't you get the id from the [heading] ? 
This way we could remove 1 extra attribute
```suggestion
       heading="Issues"
```

Reply:
That was also my first idea, but I decided against it because using an explicit attribute makes the usage of the directive more clear. Having too much alternatives and possibilities that a developer has to think about while using this directive doesn't look user friendly to me.

Reply:
> Having too much alternatives and possibilities that a developer has to think about while using this directive doesn't look user friendly to me.

But... having 2 input params vs. only 1 is not exactly that?

Reply:
> > Having too much alternatives and possibilities that a developer has to think about while using this directive doesn't look user friendly to me.
> 
> But... having 2 input params vs. only 1 is not exactly that?

There is absolutely no relation between `heading`and the functionality of the directive. Because of that i want to stay on the current implementation by using a dedicated attribute.